### DISCUSSION TITLE
Momentary Grid Disturbance - Alerts Tracked

### DISCUSSION THREAD

**Discussion by BJReplay** [jasonacox/Powerwall-Dashboard#375] [General]
Four new alerts logged last night at 23:16:
- PINV_a022_SwitchingBridgeIrrational
- PINV_a041_sensedGridDisturbance
- SYNC_a020_LoadsDropped
- SYNC_a036_LoadsDroppedLong

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/37993507/21564f1f-feeb-4fcd-8196-a5854540df91)

Lights flickered, TV and Amplifier flicked off and on, no other clock radios reset, Tesla App didn't report an outage, `https://powerwall/api/system_status/grid_faults` is empty, no obvious voltage or frequency disturbance on the charts.

Powerwall was supplying load at the time (mode set to self consumption, load was only around 500W, supplied by battery) so it was as if there was a transient grid fault and the powerwall was about to island, but didn't.  Odd.

**Comment by mcbirse** [comment]
Interesting. I have had "PINV_a022_SwitchingBridgeIrrational" and "PINV_a041_sensedGridDisturbance" previously, but it occurred when there was an actual grid outage.

- PINV_a022_SwitchingBridgeIrrational - occurred on the transition back to grid
- PINV_a041_sensedGridDisturbance - occurred some time after grid power had already returned (perhaps was not stable or storms about, I cannot remember)

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/108725631/703574c7-020c-4ca4-abb3-70705a967788)

**Reply by jasonacox** [reply]
I added some of the observations to the pypowerwall [Alerts section](https://github.com/jasonacox/pypowerwall/blob/main/README.md#tepinv---tesla-energy-powerwall-inverter) in the README.
